pages/api/universities.ts

```diff
@@ -50,8 +48,195 @@ export default async function handler(
       }
       break;
 
+    case "POST":
+      try {
+        const university: University = req.body;
+        
+        // Validate the university data using the existing schema
+        const validateUniversity = async (data: University) => {
+          const { default: Ajv } = await import("ajv");
+          const { default: addFormats } = await import("ajv-formats");
+          const ajv = new Ajv();
+          addFormats(ajv);
+          
+          const schema = {
```

Comment:
Thank you so much for this man! One thing though... we should export the schema from the script so we don't have to define it again here. 

what do you think?

Reply:
Sure, sorry for just replying now.

Any idea why the Vercel deployment is failing?

Reply:
ayy! no worries man! 

I didn't even check why the build failed. checked now and saw that it was because of an unused variable. 

![image](https://github.com/user-attachments/assets/d1ef7a25-ee6a-4025-a1af-e696441f93b9)